cmd/skuba/cluster/init.go

```diff
@@ -65,5 +67,7 @@ func NewInitCmd() *cobra.Command {
 
 	cmd.Flags().BoolVar(&initOptions.StrictCapDefaults, "strict-capability-defaults", false, "All the containers will start with CRI-O default capabilities")
 
+	cmd.Flags().StringVar(&initOptions.CniPlugin, "cni-plugin", "cilium", "Specify the CNI plugin to be used across the cluster. Valid values: cilium")
```

Comment:
Should the help sring say, 'default is 'cilium'. ?

Reply:
That would be a nice touch. I'll add that.

Reply:
It already fills the default value in "cilium", then the library "github.com/spf13/cobra" generated the default value in command help for you. So we don't need to say 'default is cilium' in help string. Otherwise, the modified current help string would be
```
Specify the CNI plugin to be used across the cluster. Default is 'cilium'. Valid values: cilium (default "cilium")
```

Reply:
Yep, just noticed this when doing some testing. I reverted this change and the help from the CLI should now look correct.